mongoose.c

```diff
@@ -10732,7 +10735,7 @@ void mg_tls_init(struct mg_connection *c, const struct mg_tls_opts *opts) {
       c->is_client ? MG_TLS_STATE_CLIENT_START : MG_TLS_STATE_SERVER_START;
 
   tls->skip_verification = opts->skip_verification;
-  tls->send.align = MG_IO_SIZE;
+  //tls->send.align = MG_IO_SIZE;
```

Comment:
Is this relevant ?
I mean, is it a desired/needed change or a debugging leftover ?

Reply:
send.align is set in net.c where all 3 buffers are initialised
https://github.com/cesanta/mongoose/blob/7923d1e930b661d699000643bc73715f6f0ebe30/src/net.c#L132